### DISCUSSION TITLE
Histogram Min/Max values do no publish via Prometheus Exporter

### DISCUSSION THREAD

**Discussion by Deniz** [open-telemetry/opentelemetry-dotnet#4090] [Q&A] [answered]
Hi,

I have been querying metrics endpoint for the Prometheus exporter by using the URL template `[host:port/metrics]` which is `localhost/metrics` by default in my setup. I could not see the min/max for my Histograms values in the response of Prometheus exporter `(OpenTelemetry.Exporter.Prometheus.AspNetCore package version 1.4.0-rc.2)`, however they exists in the Console Exporter output. I wonder if it's something with the Prometheus Exporter (not supported yet?) or something I overlooked?

Thanks,

**Comment by cijothomas** [comment, marked as answer]
Not supported in Prometheus

**Reply by Deniz** [reply]
Anything in the roadmap to support this in future that I can follow also @cijothomas?

**Reply by cijothomas** [reply]
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md#histograms-1 There is no mapping defined in the spec about what to do with Min/Max for Prometheus...
